libc/src/__support/FPUtil/multiply_add.h

```diff
@@ -39,17 +39,17 @@ multiply_add(T x, T y, T z) {
 #if defined(LIBC_TARGET_CPU_HAS_FMA)
 
 // FMA instructions are available.
-#include "FMA.h"
+#include "src/__support/FPUtil/generic_hardware/fma.h"
```

Comment:
I did this to fix a circular dependency where dyadic_float.h was including multiply_add.h -> FMA.h -> generic/FMA.h -> dyadic_float.h

Reply:
If this is the only place using it right now, maybe we can just directly use `__builtin_fma(f)` here, and adding comments about circular dependency.